task5/script.js

```diff
@@ -0,0 +1,41 @@
+const df = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [
+  {
+    to: "bookmark.html",
+    img: "1.png",
+    alt: "画像1",
+    text: "ブックマーク"
+  },
+  {
+    to: "message.html",
+    img: "2.png",
+    alt: "画像2",
+    text: "メッセージ"
+  }
+];
+
+function arrayPromise() {
+  return new Promise((resolve, reject) => {
+    resolve();
+  });
+}
```

Comment:
> 解決された値として受け取り

The problem is written to accept an array as a resolved value.
`resolve();` In this code, nothing is received, but it is resolved.
（This part is difficult, isn't it?😂）
IMO
I think this way we can receive the array as a resolved value.
```suggestion
function arrayPromise() {
  return new Promise((resolve, reject) => {
  //Let's take the value here
    resolve(listItem);
  });
}
```

Reply:
Thank you for the review.
It was difficult to understand, but I feel that I understood a little by outputting it to the "console".